xrootdfs/fs.py

```diff
@@ -145,7 +146,7 @@ def _stat_flags(self, path):
         status, stat = self.client.stat(self._p(path))
 
         if not status.ok:
```

Comment:
To match the syntax in `xrdfile.py` it should be `if not status.ok or status.error:` (ref #7)
I don't know if it's necessary to check both, as I've not found any mention of their relationship in the pyxrootd docs.